lxd/db/cluster/identities.go

```diff
@@ -288,17 +288,27 @@ type OIDCMetadata struct {
 
 // Subject returns OIDC subject from the identity metadata. The [AuthMethod] of the [Identity] must be [api.AuthenticationMethodOIDC].
 func (i Identity) Subject() (string, error) {
```

Comment:
should this be renamed to OIDCSubject now we're using an "OIDC" prefix for OIDC specific functions?

Reply:
Or should we lose this function entirely and update callers to use i.OIDCMetadata().Subject?

Reply:
Have removed the function, it is unused after this PR anyway.